app/controllers/landing_pages_controller.rb

```diff
@@ -0,0 +1,14 @@
+class LandingPagesController < ApplicationController
+  allow_unauthenticated_access only: [:show]
+
+  helper_method :show_header?
```

Comment:
What is this method for?

Reply:
`show_header?` is used to avoid rendering the header that is normally rendered on the main layout.